book/src/tutorial/high_level_rust_api.md

```diff
@@ -115,7 +113,7 @@ We need to include all `auto` files in our library.
 To do so, let's update the `src/lib.rs` file as follows:
 
 ```rust
-#![cfg_attr(feature = "dox", feature(doc_cfg))]
+#![cfg_attr(docsrs, feature(doc_cfg))]
```

Comment:
Can't this be named dox?

Reply:
Sure, we can change this. "docsrs" is just what seems to be used in other crates, I haven't seen "dox" in the wild yet. Just let me know if you prefer "dox".

Reply:
Which other crates are that?

Reply:
I guess that decides it then. Let's use the same